### PR TITLE
Newsletter

### PR DESCRIPTION
**Pull request by lithin**
### Motivation
Newsletter front-end story: create a front-end that can be then hooked into MailChimp so that users can send us emails.

### Test plan
Test in all supported browsers:
- [ ] Design looks the same as stuff on Zeplin
- [ ] When user clicks the submit button, the state of the component changes (from "before submitting" to "after submitting" and vice versa)
- [ ] On mobile, navigation on keyboard should allow user to go to the next input
- [ ] Check newsletter from accessibility point of view on mobile, desktop, and screenreader

Please note that currently the front-end does not send messages via the Contact Us service - this story is front-end only

### Pre-flight checklist
- [x] Unit tests
- [x] Reviews
  - [x] Code review
  - [x] Design review
- [x] Manual testing
  - [x] Windows 7 IE 11
  - [x] Windows 10 Edge
  - [x] Mac OS El Capitan Safari
  - [x] Mac OS El Capitan Chrome
  - [x] Mac OS El Capitan Firefox
  - [x] iOS 9 Safari
  - [x] Android 6 Chrome
- [ ] Tester approved

### CODE REVIEW

File: site/pages/home/newsletter-slice/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import AfterSignUp from './afterSignUp';
```
Comment (lpil) [inline site/pages/home/newsletter-slice/index.js]: Can we do `kebab-case` filenames for consistency? :)

File: site/pages/home/newsletter-slice/index.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import AfterSignUp from './after-sign-up';
+import BeforeSignUp from './before-sign-up';
+
+const NewsletterSlice = class extends Component {
```
Comment (lithin) [inline site/pages/home/newsletter-slice/index.js]: That's a great idea - maybe a link to the mailchimp ui instead could work.

File: site/index.ejs [hunk 3]
```diff
@@ -13,4 +13,5 @@
       <% if (typeof bodyContent !== 'undefined') { %><%= bodyContent %><% } %>
     </div>
   </body>
+  <style></style>
```
Comment (zoeabryant) [inline site/index.ejs]: remove??

File: site/pages/home/index.js [hunk 4]
```diff
@@ -1,19 +1,46 @@
...
-    <div>
-      <TopSlice />
-      <CaseStudy />
-      <Brie />
-      <TechSlice />
-      <BlogSlice />
-    </div>
-  );
-}
+const HomePage = class extends Component {
```
Comment (zoeabryant) [inline site/pages/home/index.js]: class HomePage extends

File: site/pages/home/newsletter-slice/after-sign-up.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import classnames from 'classnames/bind';
+import styles from './style.css';
+
+const cx = classnames.bind(styles);
+
+const { func } = React.PropTypes;
+
+export default class AfterSignup extends React.Component {
```
Comment (zoeabryant) [inline site/pages/home/newsletter-slice/after-sign-up.js]: Component only

File: site/pages/home/newsletter-slice/after-sign-up.js [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+            <label htmlFor="name" className={styles.formLabel}>Full name</label>
+            <input id="name" type="text" placeholder="John Smith" className={styles.formInput} />
+          </div>
+          <div>
+            <label htmlFor="name" className={styles.formLabel}>Company</label>
+            <input id="name" type="text" placeholder="Peter Pan Ltd." className={styles.formInput} />
+          </div>
+          <div>
+            <label htmlFor="name" className={styles.formLabel}>Job role</label>
+            <input id="name" type="text" placeholder="Engineer" className={styles.formInput} />
```
Comment (zoeabryant) [inline site/pages/home/newsletter-slice/after-sign-up.js]: fix for and id

File: site/pages/home/newsletter-slice/noscript.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styles from './style.css';
+
+import BeforeSignUpTitles from './before-sign-up-titles';
+
+const NoScript = () => (
+  <noscript className={styles.newsletter}>
+    <BeforeSignUpTitles />
+    <a href="http://eepurl.com/bibY5P" className={styles.link}>
+      Sing up through MailChimp
```
Comment (zoeabryant) [inline site/pages/home/newsletter-slice/noscript.js]: *sign

File: site/pages/home/newsletter-slice/before-sign-up-titles.js [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+  <div>
+    <h1 className={styles.title}>
+      Sign up to BadgerNews to hear more from us
+    </h1>
+    <p className={styles.subTitle}>
+      (every 6 weeks or so)
+    </p>
+  </div>
+);
+
```
Comment (zoeabryant) [inline site/pages/home/newsletter-slice/before-sign-up-titles.js]: move this into before sign up - same for noscript

File: site/pages/home/newsletter-slice/style.css [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+  }
+
+  .formInput {
+    width: 400px;
+  }
+
+  .submitButton {
+    display: inline-block;
+    position: relative;
+    top: 1px;
```
Comment (zoeabryant) [inline site/pages/home/newsletter-slice/style.css]: vertical align?

File: site/pages/home/test.js [hunk 10]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Home from '.';
+import AfterSignUp from './newsletter-slice/after-sign-up';
+import BeforeSignUp from './newsletter-slice/before-sign-up';
+
+describe('Newsletter slice', () => {
```
Comment (zoeabryant) [inline site/pages/home/test.js]: wrap in HomePage thingy

**Pull request merged**